tests/test_browser.py

```diff
@@ -2730,6 +2730,11 @@ def test_sdl2_image_prepare(self):
     shutil.copyfile(path_from_root('tests', 'screenshot.jpg'), os.path.join(self.get_dir(), 'screenshot.not'))
     self.btest('sdl2_image_prepare.c', reference='screenshot.jpg', args=['--preload-file', 'screenshot.not', '-s', 'USE_SDL=2', '-s', 'USE_SDL_IMAGE=2'])
 
+  def test_sdl_image_prepare_data(self):
```

Comment:
The name here should have `sdl2`. Actually, it's funny, due to how python works, it allows the same name twice and leaves just the last one, so this could have overridden the old test (or vice versa) ;)

Also, please add yourself to AUTHORS.

Thanks!